lighthouse/helpers/plates.py

```diff
@@ -470,21 +495,174 @@ def create_cherrypicked_post_body(barcode: str, samples: List[Dict[str, Any]]) -
         content = {}
 
         if "control" in sample:
+            content["supplier_name"] = sample["supplier_name"]
             content["control"] = sample["control"]
             content["control_type"] = sample["control_type"]
+            content["uuid"] = sample["uuid"]
         else:
             content["name"] = sample["name"]
             content["phenotype"] = sample["phenotype"]
             content["supplier_name"] = sample["supplier_name"]
             content["sample_description"] = sample["sample_description"]
+            content["uuid"] = sample["uuid"]
 
         wells_content[sample["coordinate"]] = {"content": content}
 
+    subjects = []
+    subjects.append(robot_subject(robot_serial_number))
+    subjects.extend(source_plate_subjects(plate_id_mappings))
+    subjects.extend(sample_subjects(samples))
+
+    events = [
+        {
+            "event": {
```

Comment:
Do you also need a `"uuid"` and `"occured_at"` field in this event message like the ones sent in the plate-event/create endpoint? Or is this something SS provides, and If so might be worth commenting why?

Reply:
Yes, this is created in SS. That is the uuid and time the record is created in database